modules/services/window-managers/i3-sway/sway.nix

```diff
@@ -335,7 +328,7 @@ in {
 
     package = mkOption {
       type = with types; nullOr package;
-      default = defaultSwayPackage;
+      default = pkgs.sway;
       defaultText = literalExpression "${pkgs.sway}";
```

Comment:
While you're at it
```suggestion
      defaultText = literalExpression "pkgs.sway";
```

Reply:
I've used `swayfx` to show something different than the default.

Reply:
also it should have been sway-unwrapped anyway

Reply:
The `defaultText` should show what the `default` value is. Use `example` for an example.